cmd/release-controller/http.go

```diff
@@ -215,13 +213,6 @@ const releaseDashboardPageHtml = `
 </div>
 `
 
-var (
```

Comment:
Can you do the move of this code in one commit, and the changes for your new stuff in another?

Reply:
Sure thing!